src/openshift/cluster.ts

```diff
@@ -500,6 +500,16 @@ export class Cluster extends OpenShiftItem {
         );
     }
 
+    static async validateLoginToken(userClusterUrl: string, userToken: string): Promise<boolean> {
+        try {
+            await Cluster.cli.executeTool(Command.odoLoginWithToken(userClusterUrl, userToken.trim()), undefined, true);
```

Comment:
I'm not sure if there is a better way to check whether the token is valid or not

Reply:
I think the regular expression check should be enough. I think that logging in and out of the cluster isn't necessary, and might slow down the extension. If the contents of the clipboard match the regex, but the token isn't valid, then we could show a popup when they hit the "Login to DevSandbox" button